### PR TITLE
Meenu/fix: Integrate chips component bug fixes

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Trade page improvements:

- Fixed bottom sheet not displaying selected info(This is a temp fix as action sheet will be replaced from quill-design)

- Fixed action sheet appearing on large screens on zoom

- Fixed chips content appearing on two lines

### CODE REVIEW

File: libs/templates/src/lib/trade/table/trading-spec-table.tsx [hunk 1]
```diff
@@ -220,11 +220,61 @@ const TradingSpecTable = () => {
...
+                setShowBottomSheet(false);
+              }}
+            >
+              {selectedInfo === 'additionalInfo' ? (
+                <StandaloneCircleDotFillIcon fill="black" iconSize="md" />
+              ) : (
+                <StandaloneCircleRegularIcon fill="black" iconSize="md" />
+              )}
+
+              <Text size="md" className="text-typography-default">
```
Comment (jia-deriv) [inline libs/templates/src/lib/trade/table/trading-spec-table.tsx:273]: can you check if you remove `text-typography-default` will affect anything? By default this class will apply automatically

File: libs/templates/src/lib/trade/table/trading-spec-table.tsx [hunk 2]
```diff
@@ -220,11 +220,61 @@ const TradingSpecTable = () => {
...
           }
         >
-          <ActionSheetBottom
-            onChangeChips={handleChipsData}
-            onChangeBottomSheet={handleBottomSheet}
-            onChangeSelectedInfo={handleChipsInfo}
-          />
+          <div className=" flex flex-col p-general-md">
+            <div
+              className="flex flex-row items-center gap-gap-md px-general-md py-gap-md"
```
Comment (jia-deriv) [inline libs/templates/src/lib/trade/table/trading-spec-table.tsx:225]: please use py-general-[value]
cc: @Nuzhy-Deriv

File: libs/templates/src/lib/trade/table/trading-spec-table.tsx [hunk 3]
```diff
@@ -100,16 +98,18 @@ const TradingSpecTable = () => {
           <LabelPairedEllipsisVerticalBoldIcon
             fill="black"
             iconSize="md"
-            className="block md:hidden lg:hidden"
+            className="min-sm:block max-lg:block md:hidden lg:hidden"
```
Comment (Nuzhy-Deriv) [inline libs/templates/src/lib/trade/table/trading-spec-table.tsx:101]: `min-sm:block` is not valid in tailwind.
```suggestion
            className="block md:hidden"
```

File: libs/templates/src/lib/trade/table/trading-spec-table.tsx [hunk 4]
```diff
@@ -100,16 +98,18 @@ const TradingSpecTable = () => {
...
             onClick={() => {
               setShowBottomSheet(true);
             }}
           />
 
-          <Chips
-            onChangeChips={handleChipsData}
-            onChangeSelectedInfo={handleChipsInfo}
-          />
+          <div className="min-sm:hidden w-max max-lg:hidden md:block lg:block">
```
Comment (Nuzhy-Deriv) [inline libs/templates/src/lib/trade/table/trading-spec-table.tsx:107]:
```suggestion
          <div className="hidden w-max md:block">
```

File: libs/templates/src/lib/trade/chips/index.tsx [hunk 5]
```diff
@@ -14,37 +15,34 @@ const Chips = ({ onChangeChips, onChangeSelectedInfo }: ChipsProps) => {
...
-        className="hidden md:block lg:block "
         onChipSelect={() => {
           onChangeChips(additionalInfo.data);
           onChangeSelectedInfo('additionalInfo');
           setSelectedInfo('additionalInfo');
         }}
         selected={selectedInfo === 'additionalInfo'}
       >
-        Additional info
+        <div className="w-max">Additional info</div>
```
Comment (prince-deriv) [inline libs/templates/src/lib/trade/chips/index.tsx:45]: Why do we need this?

Comment (meenakshi-deriv) [reply libs/templates/src/lib/trade/chips/index.tsx:45]: on some responsive views the content inside chips is taking two lines. This should be fixed from quill-design.

File: libs/templates/src/lib/trade/table/trading-spec-table.tsx (hunk 1 above)
Comment (meenakshi-deriv) [reply libs/templates/src/lib/trade/table/trading-spec-table.tsx:273]: yes can remove this. Thanks @jia-deriv

**Review by jia-deriv — APPROVED** [review]

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**